packages/grafana-ui/src/components/Collapse/Collapse.tsx

```diff
@@ -105,7 +105,7 @@ export interface Props {
   className?: string;
 }
 
-export const ControlledCollapse: FunctionComponent<Props> = ({ isOpen, onToggle, ...otherProps }) => {
+export const ControlledCollapse = ({ isOpen, onToggle, ...otherProps }: React.PropsWithChildren<Props>) => {
```

Comment:
Alternatively, we could just type `children: React.ReactNode` in`Props`. At the risk of bikeshedding this, might be worth checking with broader group for what pattern we want to prefer.

Reply:
agreed, think it's worth discussing in a frontend weekly. don't think it needs to hold out this specific PR since we already have both throughout the codebase (😅).

i would propose using `React.PropsWithChildren` more tbh. it comes from react, so if they ever change the children type (extremely unlikely i know) it would be a no-op. and it removes any chance of an incorrect type being set (what was it again... was it `React.ReactNode`, `React.ReactElement` or `JSX.Element`? 🤔)

Reply:
it's just React.ReactNode. always React.ReactNode 😅 

I think.